README.md

```diff
@@ -1148,9 +1148,9 @@ car.save();
 ```javascript
 class Car {
   constructor() {
-    this.make = 'Honda';
-    this.model = 'Accord';
-    this.color = 'white';
+    this.make = '';
```

Comment:
Because it has no sense. For example you add new Honda to user and then you set it to Ford?